packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -436,7 +436,8 @@ export const EVENT_TRIGGERS = {
   MEDIA_QUALITY: 'media:quality',
   MEETINGS_NETWORK_DISCONNECTED: 'network:disconnected',
   MEETINGS_NETWORK_CONNECTED: 'network:connected',
-  MEETING_SELF_LEFT: 'meeting:self:left'
+  MEETING_SELF_LEFT: 'meeting:self:left',
+  NETWORK_QUALITY: 'network:quality'
```

Comment:
nice

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-246903 - jira task for downlink